spec/unit/service_spec.rb

```diff
@@ -335,6 +335,21 @@
       end
     end
 
+    context 'service start successful for startup type is disable' do
+      before do
+        allow(instance).to receive(%x{powershell.exe -Command "Set-Service -Name 'chef-client' -StartupType Disabled"}).and_return(true)
```

Comment:
This line doesn't execute as we are not calling `%x{powershell.exe -Command "Set-Service -Name 'chef-client' -StartupType Disabled"}` in the code.